### DISCUSSION TITLE
谜一样的1028，抢到了还会出现

### DISCUSSION THREAD

**Discussion by Aellyt** [Ljzd-PRO/Mys_Goods_Tool#158] [Show and tell]
![5f493ab2e8533c8dd3a30f6b6a7e61c](https://github.com/Ljzd-PRO/Mys_Goods_Tool/assets/83526546/b94531bd-fab5-4518-b0f3-f201b86e9d18)

**Comment by Satoing** [comment]
请问大佬你是怎么运行的？你之前提交的版本还没有发布release，我直接运行 `__init__.py` 开始抢的时间比整点晚了10秒。

**Comment by Aellyt** [comment]
我用自己写的，js的脚本，丢青龙面板跑的

**Comment by dmhfd** [comment]
可能又改了，我之前抢到再调用只会报兑换超限，不会1028

**Reply by Aellyt** [reply]
但是再跑就是超限，只有兑换的那次这样
